ivy_tests/test_ivy/helpers/hypothesis_helpers/array_helpers.py

```diff
@@ -1047,7 +1047,6 @@ def array_indices_put_along_axis(
 
     Parameters
     ----------
-
```

Comment:
I think these are not related to the function you are implementing, so I you can revert it back :slightly_smiling_face: